kind/README.md

```diff
@@ -0,0 +1,50 @@
+# Redpanda on KinD
+
+## TLDR
+
+```sh
+./make.sh
+```
+
+## Details
+
+### Local Registry
+
+There is a [local registry](https://kind.sigs.k8s.io/docs/user/local-registry/).
+
+The registry can be used like this.
+
+1. First we'll pull an image `docker pull vectorized/redpanda:latest`
+2. Then we'll tag the image to use the local registry `docker tag vectorized/redpanda:latest localhost:5000/redpanda:latest`
+3. Then we'll push it to the registry `docker push localhost:5000/redpanda:latest`
+4. And now we can use the image `kubectl create deployment hello-server --image=localhost:5000/redpanda:latest`
+
+If you build your own image and tag it like `localhost:5000/redpanda:latest` and then use it in kubernetes as `localhost:5000/redpanda:latest`.
+
+### LoadBalancer
+
+[metallb](https://metallb.universe.tf/) is used.
+
+This works on Ubuntu - YMMV.
+
+Until [multiple listeners are supported](https://github.com/vectorizedio/redpanda/discussions/257), `/etc/hosts` needs to be modified.
+
+Installation will describe how.
+
+### Redpanda
+
+3-node cluster.
+
+Each node has a PVC.
+
+Testing a rolling restart:
+
+```sh
+kubectl -n redpanda rollout restart statefulset redpanda
+```
+
+### Accessing Kubernetes
+
+I'm not a fan of [kubernetes-dashboard](https://github.com/kubernetes/dashboard#kubernetes-dashboard) and it's a pain to set up.
```

Comment:
Maybe project [octant](https://octant.dev/) can be listed hear too.

Reply:
These types of opinions probably should be in this repo. I'll remove it.